INN/umbrella-currentorg
# Hide extra image radio button for thumbnail

## Changes

This pull request makes the following changes:

- Hides the `Set this image as the listing thumbnail.` radio button for listing images when creating a listing.

Before:
![Screen Shot 2019-07-08 at 1 12 24 PM](https://user-images.githubusercontent.com/18353636/60829072-11ebd780-a182-11e9-9cc0-820af0fe9f57.png)

After:
![Screen Shot 2019-07-08 at 1 12 34 PM](https://user-images.githubusercontent.com/18353636/60829071-11ebd780-a182-11e9-8cfe-3a5d430b9f1b.png)

## Why



For #74 

## Testing/Questions

Features that this PR affects:

- Create a listing page


Questions that need to be answered before merging:

- [ ] How's it look?

Steps to test this PR:

1. View the create a listing page
2. Upload multiple images
3. Make sure there is no radio button
